### PR TITLE
[Docs] Replaced references to Unity Buy SDK with Shopify SDK for Unity

### PR DESCRIPTION
**Pull request by sleroux**

### CODE REVIEW

File: BUILDDETAILS.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Build Details
 
-The Unity Buy SDK uses Unity features to modify certain files during the build phase when building for a specific platform.
+The Shopify SDK for Unity uses Unity features to modify certain files during the build phase when building for a specific platform.
```
Comment (mikkoh) [inline BUILDDETAILS.md]: I wonder if here we should just say. `The SDK`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ To check your the project's dependencies run:
 $ scripts/check_setup.sh
 ```
 
-## Building the Unity Buy SDK
+## Building the Shopify SDK for Unity
 
-Before you can start developing on the SDK you'll need to run the code generator. Under the hood, the Unity Buy SDK works on top of GraphQL. In order to query Shopify, we generate a GraphQL client based on the [Storefront API](https://help.shopify.com/api-storefront-api) introspection schema.
+Before you can start developing on the SDK you'll need to run the code generator. Under the hood, the Shopify SDK for Unity works on top of GraphQL. In order to query Shopify, we generate a GraphQL client based on the [Storefront API](https://help.shopify.com/api-storefront-api) introspection schema.
```
Comment (mikkoh) [inline CONTRIBUTING.md]: I know this is outside of the scope of this PR but I wonder if we should say something like.

> If you do not want to download a Unity Package you have the option of building source that you can import into your project.

File: Doxyfile [hunk 3]
```diff
@@ -32,7 +32,7 @@ DOXYFILE_ENCODING      = UTF-8
 # title of most generated pages and in a few other places.
 # The default value is: My Project.
 
-PROJECT_NAME           = "Unity Buy SDK"
+PROJECT_NAME           = "Shopify SDK for Unity"
```
Comment (mikkoh) [inline Doxyfile]: Where is this used? Are we sure everything is safe with this change?

Comment (sleroux) [reply Doxyfile]: Doxygen uses this as the title mostly from what I've seen.

**Comment by mikkoh** [comment]
Can you revert generating the docs so this PR is easier to review?

**Comment by sleroux** [comment]
Reverted. CI might fail just because this is opened against master for now and the CI fixes are on beta.

**Comment by sleroux** [comment]
@mikkoh Should be good/rebased for landing 👍?

Before I land, should I run the doxygen update and commit that in?

**Pull request merged**